### PR TITLE
[stable16] Bump nextcloud-vue-collections to 0.4.2

### PR DESCRIPTION
**Pull request by juliusknorr**
### Changelog:

- Force defaultHtml setting of v-tooltip to be disabled
- Remove duplicate slash from delete url

### CODE REVIEW

**Review by rullzer — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Comment by skjnldsv** [comment]
> /compile amend /

doesn't work on 16

**Comment by juliushaertl** [comment]
Ah damn, I was unable to compile it locally because of the outdated node-sass, let me try again then
